app/views/maintaining/dashboard/show.html.haml

```diff
@@ -9,6 +9,11 @@
           = link_to "All Submissions", maintaining_submissions_path
         %li
           = link_to "All Representations", maintaining_exercise_representations_path
+        %li
+          = link_to "Site Updates", maintaining_site_updates_path
+        - if current_user.admin?
+          %li
+            = link_to "Contributor Teams", maintaining_contributor_teams_path
```

Comment:
- We may want all maintainers to be able to see the data, and only admins to be able to edit it.
- If we only want admins to see it, we should move this whole section into an `app/views/admin` namespace.

I'm not sure which is right.

Reply:
> We may want all maintainers to be able to see the data, and only admins to be able to edit it.

This might be the nicest option. I'll update